tests/test_crystal_ball.py

```diff
@@ -1,13 +1,18 @@
 """Module for testing the htcrystalball module."""
-from htcrystalball import examine, collect, utils
-from os import path
-from pytest import fixture, raises as praises
 import argparse
 import json
+from os import path
+from pytest import fixture, raises as praises
+from htcrystalball import examine, collect, utils
 
 
 @fixture
 def root_dir():
+    """
+    Defines the absolute path to this file.
+    Returns:
+
+    """
     return path.dirname(path.abspath(__file__))
```

Comment:
This function returns a string containing the directory name of the absolute path of the currently opened file (i.e., if executed in `/path/to/file.txt` it would return `/path/to`). I don't think this needs a returns value, as both summary and the one line of code in the function are descriptive enough of what it does. You could change line 10 to `def root_dir() -> str:` though.